crates/matrix-sdk/src/account.rs

```diff
@@ -1059,12 +1059,16 @@ impl Account {
         // We need to get the initial value of the media preview config event
         // we do this after creating the observers to make sure that we don't
         // create a race condition
-        let initial_value = self.get_media_preview_config_event_content().await?;
+        let initial_value = self.get_stored_media_preview_config_event_content().await?;
 
         Ok((initial_value, result_stream))
     }
 
-    async fn get_media_preview_config_event_content(
+    /// Get the media preview configuration event content from the server.
+    ///
+    /// If no value was found in either the stable and unstable type, the
+    /// default configuration will be returned.
+    pub async fn get_media_preview_config_event_content(
```

Comment:
Now that we have two methods, let's call this one `fetch_...()` and the one that access the store `get()`, to avoid the overly long name.